numpy/lib/tests/test_function_base.py

```diff
@@ -3048,7 +3053,7 @@ def test_lower_higher(self, dtype):
         assert_equal(np.percentile(np.arange(10, dtype=dtype), 50,
                                    method='higher'), 5)
 
-    @pytest.mark.parametrize("dtype", TYPE_CODES)
+    @pytest.mark.parametrize("dtype", np.typecodes["Float"])
```

Comment:
Just passing through, missed that before.  You removed Complex above already, using float here removes integers and object here limits the test (not that it looks like the most thorough one.

Reply:
Fixed.